encord/objects/classification_instance.py

```diff
@@ -330,11 +359,22 @@ def remove_from_frames(self, frames: Frames) -> None:
         if self._parent:
             self._parent._remove_frames_from_classification(self, frames)
 
-    def get_annotations(self) -> List[Annotation]:
+    # TODO: Need to deprecate this old Annotation
+    def get_annotations(self) -> Union[List[Annotation], List[_ClassificationAnnotation]]:
```

Comment:
I would really hold off on modifying this method in this PR since it's part of the public API. What do you think of leaving out the space support here for this PR? The new annotations would still be accessible through the space, and we don't risk modifying the public API.